bikeshed/__init__.py

```diff
@@ -796,9 +796,21 @@ def decorateAutolink(doc, el, linkType, linkText):
 
 def processIssues(doc):
     import hashlib
-    # Add an auto-genned and stable-against-changes-elsewhere id to all issues.
+    # Add an auto-genned and stable-against-changes-elsewhere id to all issues, and
+    # link to remote issues if possible:
     for el in findAll(".issue:not([id])", doc):
         el.set('id', "issue-"+hashContents(el))
+        remoteIssueID = el.get('data-remote-issue-id')
+        if remoteIssueID:
+            del el.attrib['data-remote-issue-id']
+            githubMatch = re.match(r"\s*([\w\-_]+)/([^\w\-_]+)#(\d+)\s*$", remoteIssueID)
```

Comment:
`r"\s*([\w\-_]+)/([^\w\-_]+)#(\d+)\s*$"` I think you messed up that middle one; you almost certainly don't mean to negate it.  Also, `_` is already part of `\w` so you can drop it, and you don't need to escape `-` if you put it last in the character class.